Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -151,7 +151,16 @@ file that was distributed with this source code.
         var defaults = {% include 'SonataDoctrinePHPCRAdminBundle:Tree:routing_defaults.html.twig' with {'routing_defaults': routing_defaults} %};
 
         $('#{{ id }}-tree-selector').cmfTree({
-            data_url: Routing.generate('_cmf_tree_{{ tree.alias }}_children', defaults),
+            request: {
+                load: function (nodePath) {
+                    return {
+                        url: Routing.generate('_cmf_resource', jQuery.extend(defaults, {
```

Comment:
I would really like to remove the fos js routing .. its needless overhead imo, we can generate the URL using ``path('_cmf_resource', array('repositoryName' => '%repo_name%', 'path' => '%path%'))`` and then use string substitution in the js.